tests/Containerfile

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/centos/centos:latest
+
+RUN true \
```

Comment:
Why do we need `true` as first command here?

Reply:
just a convention, just like the last `true`. It makes it possible to add/modify commands and still have reasonable `git blame` results.